go.mod

```diff
@@ -91,4 +91,4 @@ require (
 )
 
 // replace github.com/gophercloud/gophercloud => ./external/gophercloud
-replace github.com/gophercloud/gophercloud => github.com/Wind-River/gophercloud v0.0.0-20230119154844-fa8a1286cb56
+replace github.com/gophercloud/gophercloud => github.com/sselvara1/gophercloud v0.0.0-20230223101244-77c13c232f8c
```

Comment:
Before merging the code, don't forget to change back the gophercloud commit to the commit in repo github.com/Wind-River/gophercloud

Reply:
sure

Reply:
We don't need to put it back to "Wind-River"?

Reply:
Yes, we do need to put it back, good catch, thanks!